### PR TITLE
[WIP] Guest page

### PR DESCRIPTION
**Pull request by waliens**
BGWE/BoardgameWE#118

### CODE REVIEW

**Comment by elodieburtin** [comment]
:heart_eyes:

File: boardgame-ui/src/pages/GuestHomePage.vue [hunk 1]
```diff
@@ -15,16 +15,123 @@
         </div>
       </section>
 
-      Lorem ipsum..
+      <div class="container">
+        <section class="content app-description">
+          <h2>Connecting board game players</h2> 
+          <p><i18n path="app.appName"/> is an application for those who like playing to board games. It offers several features for connecting players and enhancing the board game playing experience.</p>
```
Comment (FServais) [inline boardgame-ui/src/pages/GuestHomePage.vue]: * "playing board games" 😁

Comment (waliens) [reply boardgame-ui/src/pages/GuestHomePage.vue]: Oops, actually had it right the first time..

**Comment by FServais** [comment]
@waliens, can you fix the conflicts? IMO good to be merged after :)

**Comment by waliens** [comment]
Nope nope nope, there was still some work to do on this ! Text must be internationalized !! @FServais @hjwk

**Pull request merged**